runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7886,7 +7908,10 @@ TR::CompilationInfoPerThreadBase::compile(J9VMThread * vmThread,
             static_cast<TR::SegmentAllocator &>(debugSegmentProvider) :
             static_cast<TR::SegmentAllocator &>(defaultSegmentProvider);
       TR::Region dispatchRegion(regionSegmentProvider, rawAllocator);
-      TR_Memory trMemory(*_compInfo.persistentMemory(), dispatchRegion);
+
+      // Initialize trMemory with per-client persistent memory, since
```

Comment:
Per-client persistent memory is only used for JITServer, so let's adjust the comment to reflect that

Reply:
Done.